src/patterns/error-validation/examples/errors-proto/errors/index.njk

```diff
@@ -299,6 +308,37 @@ layout: none
                 }
             })
         }}
+
+        {{
+            onsDuration({
+                "label": "Duration",
+                "id": "address-duration",
+                "dontWrap": true,
+                "error": {
+                    "text": "Enter a number that is less that 60",
+                    "id": "duration-error"
+                },
+                "field1": {
+                    "id": "address-duration-hours",
+                    "name": "address-duration-hours",
+                    "label": "Hours",
+                    "attributes": {
+                        "min": 0,
+                        "max": 100
+                    }
+                },
+                "field2": {
+                    "id": "address-duration-minutes",
+                    "name": "address-duration-minutes",
+                    "label": "Minutes",
+                    "value": "60"
```

Comment:
Missing comma is breaking the template, but will be fine if you remove the following invalid attributes